packages/expo-live-photo/src/LivePhoto.types.ts

```diff
@@ -0,0 +1,121 @@
+import { ViewProps } from 'react-native';
+
+export interface LivePhotoViewProps extends ViewProps {
+  /**
+   * The live photo asset to display.
+   */
+  source?: LivePhotoAsset | null;
+  /**
+   * Determines whether the live photo should also play audio.
+   *
+   * @default true
+   */
+  isMuted?: boolean;
+
+  /**
+   * Determines how the image should be scaled to fit the container.
+   * - `'contain'` - Scales the image so that its larger dimension fits the target size.
+   * - `'cover'` - Scales the image so that it completely fills the target size.
+   *
+   * @default 'contain'
+   */
+  contentFit?: ContentFit;
+
+  /**
+   * Determines whether the default iOS gesture recognizer should be used.
+   * When true the playback will start if the user presses and holds on the LivePhotoView.
+   *
+   * @default true
+   */
+  useDefaultGestureRecognizer?: boolean;
+
+  /**
+   * Called when the playback starts.
+   */
+  onPlaybackStart?: () => void;
+
+  /**
+   * Called when the playback stops.
+   */
+  onPlaybackStop?: () => void;
+
+  /**
+   * Called when the live photo starts being loaded.
```

Comment:
```suggestion
   * Called when the live photo starts loading.
```

I think we can avoid the passive voice usage here.